ui/src/app/components/webcomponents/action-element-suggestion.component.ts

```diff
@@ -78,14 +79,14 @@ export class ActionElementSuggestionComponent implements OnInit {
     pageable.pageSize = 50;
     let searchName = '';
     if (term) {
-      searchName = ",name:*" + term + "*";
+      searchName = ",name:*" + encodeURIComponent(new ElementFilter().byPassSpecialCharacters(term)) + "*";
       this.isQueryBased = true;
```

Comment:
@nixonakash01 Did you find out why the issue was not reproduced consistently ?

Reply:
@tarun-testsigma this issues is reproduced if the element has special characters in them. special characters in elements is created automatically by recorder. this fix is already implemented in cloud. and i verified that it works.